### ISSUE TITLE
Issue with client.apps being undefined

### ISSUE THREAD

**Issue by BrandonPessman** [ceifa/steamworks.js#57]
Hi there!
I'm currently running into an issue with trying to access the functions in client.apps. For some reason, it doesn't seem to include those functions when I print it out, nor when I try them. I just get a client.apps is undefined. Here is an example of what I am trying to do. 

```js
  const steamworks = require('steamworks.js')
  let client

  try {
    client = steamworks.init(2233750)
  } catch (err) {
    console.log(err)
    client = null
  }

  console.log(client.apps.appBuildId())
  console.log('client', client)
```

As you can see in the image below, I'm not seeing apps either in that list. 

<img width="918" alt="Screen Shot 2022-12-24 at 1 42 09 AM" src="https://user-images.githubusercontent.com/34040658/209428139-17f46a8a-b08b-4d93-8cc7-d2c7054c6cb2.png">

Here is the error:
<img width="1130" alt="Screen Shot 2022-12-24 at 1 44 50 AM" src="https://user-images.githubusercontent.com/34040658/209428213-d499dc7a-2546-4d8d-b8f2-d2ea53c598bc.png">

**Version:** 0.0.25
**OS:** Mac

Any help here? I can't seem to find documentation on it, so I can't tell if it's a bug or if it's an issue on my end. I typically think it's an issue on my end. I kind of wonder if it's due to me being on a Mac. Maybe the dist for Mac wasn't built yet?

**Comment by ceifa** [comment]
The MacOS build for Apple Silicon processors it's in an older version of `steamworks.js`.
Related #17

**Comment by BrandonPessman** [comment]
@ceifa Ah! I see! Is there any way I can build the dist for it myself? I have a m1 Mac and don’t mind running a build for it and putting the dist in manually. You have some great work here.

**Comment by BrandonPessman** [comment]
I've gone ahead and completed a build. I was able to figure it out. Pull request has been added if you'd like to include an updated build for macs.

**Comment by ceifa** [comment]
Thanks for it! Just released the version 0.0.26 for you.
Merry christmas!
